src/Behat/Gherkin/Node/OutlineNode.php

```diff
@@ -52,22 +52,22 @@ class OutlineNode implements ScenarioInterface
      * @param null|string      $title
      * @param string[]         $tags
      * @param StepNode[]       $steps
-     * @param ExampleTableNode $table
+     * @param ExampleTableNode[] $tables
      * @param string           $keyword
      * @param integer          $line
      */
     public function __construct(
         $title,
         array $tags,
         array $steps,
-        ExampleTableNode $table,
+        array $tables,
```

Comment:
This is a backwards compatibility break. Unless you are willing to wait till next Gherkin major, which might never happen on this codebase, you'd better approach this differently.